tests/velero/restore.go

```diff
@@ -78,35 +78,32 @@ func (c *ClientSet) GetScheduledBackups(schedule string) ([]string, error) {
 	return bkplist, nil
 }
 
-// CreateRestoreFromSchedule restore from given schedule's n'th backup for ns Namespace
-func (c *ClientSet) CreateRestoreFromSchedule(ns, targetedNs, schedule string, n int) (v1.RestorePhase, error) {
-	var status v1.RestorePhase
-	var err error
+// CreateRestore create restore from given backup/schedule for ns Namespace to targetedNs
+// Arguments:
+// - ns : source namespace or namespace from backup
+// - targetedNs : targeted namespace for namespace 'ns'
+// - backup : name of the backup, from which restore will happen
+// - schedule : name of schedule, from which restore should happen. If mentioned, backup should be empty
+func (c *ClientSet) CreateRestore(ns, targetedNs, backup, schedule string) (v1.RestorePhase, error) {
```

Comment:
I don't see the change in caller.. will that be in diff repo?

Reply:
It is in same PR. relevant test case `Restore from scheduled backup`